src/menus.js

```diff
@@ -75,6 +75,13 @@ var menus = {
 			description: "Manages storage of previously created or opened pictures.",
 		},
 		$MenuBar.DIVIDER,
+		{
+			item: "Upload to &Imgur",
+			shortcut: "Ctrl+I",
```

Comment:
Ctrl+I is already used for Image > Invert, and it shouldn't show a shortcut if it's not hooked up to work; you can just leave off the `shortcut` property. You can keep the `&` because that's a separate thing, and it looks like it doesn't 
 conflict with anything in that menu. (Although I'm realizing I accidentally duplicated the `&L` used by `Set As Wa&llpaper (Centered)`.)
"to" should be capitalized, because that's the scheme as far as I can tell, even small words like "As" are capitalized.

Reply:
Also, I would move it up to a section together with Load From URL, below Save and Save As. Maybe move Manage Storage up there too, but still in its own section. That way things that are conceptually related are close together, and it sort of flows, Just Filesystemy Things | Files And The Internet | Files Stored Locally | Other Stuff | Other Stuff... - and the Other Stuff is also less important, so it's good if it's further down.